k4FWCore/src/PodioDataSvc.cpp

```diff
@@ -62,6 +63,22 @@ StatusCode PodioDataSvc::initialize() {
     m_metadataframe = podio::Frame();
   }
 
+  IProperty* property;
+  auto       sc = service("ApplicationMgr", property);
+  if (sc == StatusCode::FAILURE) {
+    error() << "Could not get ApplicationMgr properties" << std::endl;
+  }
+  Gaudi::Property<int> evtMax;
+  evtMax.assign(property->getProperty("EvtMax"));
+  m_requestedEventMax = evtMax;
+  m_requestedEventMax -= m_1stEvtEntry;
```

Comment:
Is going through a dedictated `Gaudi::Property` necessary? Could we not simply do `property->getProperty("EvtMax")` and assign that directly to `m_requestedEventMax`? Is this mainly to get the correct (implicit) conversion to `int`? Can we trigger an explicit conversion from the original property and skip the intermediate Property?

Reply:
I could do this instead if you prefer
```
m_requestedEventMax = dynamic_cast<const Gaudi::Property<int>&>(property->getProperty("EvtMax"));
```

Reply:
Eh, not entirely better ;) I think keeping the current version is fine.

Reply:
Another alternative is 
``` cpp
m_requestedEventMax = std::stoi(property->getProperty("EvtMax").toString());
```

Reply:
that looks to me like the least clean way to do this... :D